Logger/Logger/Logger/Logger.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    internal class Logger
+    {
+        private static Logger instance;
+        private List<string> logs;
+        private string logFilePath;
+
+        private Logger() { }
+
+        public static Logger GetInstance()
+        {
+            if(instance == null)
+            {
+                instance = new Logger();
+            }
+            return instance;
+        }
+        public void Initialize(string filePath)
```

Comment:
You kinda rely on users of this class to know that they have to call this method first.